static/js/settings_org.js

```diff
@@ -973,6 +975,14 @@ function populate_data_for_request(subsection, for_realm_default_settings) {
                 } else {
                     [, property_name] = /^id_realm_(.*)$/.exec($input_elem.attr("id"));
                 }
+
+                if (property_name === "stream_privacy") {
+                    Object.assign(
```

Comment:
Should this be using the spread operator instead?

Reply:
Can be done that way too. Updated.